docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1591,6 +1588,61 @@ This template extension method makes it possible to render the following templat
 ----
 <1> `item` is resolved to an instance of `org.acme.Item`.
 
+However, it is possible to specify the matching name with `matchName()`.
+
+.`TemplateExtension#matchName()` Example
+[source,java]
+----
+@TemplateExtension(matchName = "discounted")
+static BigDecimal discountedPrice(Item item) {
+   // this method matches {item.discounted} iff "item" resolves to an object assignable to "Item"
```

Comment:
```suggestion
   // this method matches {item.discounted} if "item" resolves to an object assignable to "Item"
```

Reply:
Well, I meant https://en.wikipedia.org/wiki/IFF...

Reply:
That looked like a typo to me 😀 

I'd suggest using the longer `if (and only if)` for clarity